### PR TITLE
Add install instructions

### PR DESCRIPTION
**Pull request by Alok**
Having to bury into the 3rd or so page of the docs just to install `hypothesis` is annoying, especially if you just want to be sure that `pip install hypothesis` is the right command.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -26,6 +26,15 @@ Hypothesis is extremely practical and advances the state of the art of
...
 you're not using Hypothesis to test your project then you're missing out.
 
+------------------------
+Quick Start/Installation
+------------------------
+If you just want to get started:
+
+.. code-block:: sh
+pip install hypothesis
+.. code-block::
```
Comment (alexwlchan) [inline README.rst]: I think the syntax is a bit off here. Here’s what [the rendered version](https://github.com/Alok/hypothesis-python/blob/patch-1/README.rst) looks like in my web browser:

screen shot 2016-11-26 at 07 16 05

Looking at the other examples in the README, I think you want something more like:

```rst
.. code-block:: sh

  pip install hypothesis
```

**Comment by alexwlchan** [comment]
Ping @DRMacIver – now tests are passing, are you okay to merge this?

(I realise the README is the public face of the project, so I want to double-check with you before editing it.)

**Comment by DRMacIver** [comment]
👍 from me.

It might be worth also adding a link to the quickstart in the documentation below that? But this is fine to merge as is.

**Comment by alexwlchan** [comment]
Okay, I’ll merge this as-is. Thanks @Alok!

**Pull request merged**